README.md

```diff
@@ -23,15 +19,42 @@ PHP client library for [Coveralls](https://coveralls.io).
 
 # Installation
 
+tl;dr
+Try the phar file first, come back if it doesn't work for you.
```

Comment:
Both line are pointless. `tldr` didn't bring any value as full version isn't long.
Suggesting to install phar first isn't nice - leave the decision to users. One would prefer phar, one not.